include/mbgl/style/sources/vector_source.hpp

```diff
@@ -26,6 +26,14 @@ class VectorSource final : public Source {
 
     void loadDescription(FileSource&) final;
 
+    /// @brief Gets the tile urls for this vector source.
+    /// @return std::vector<std::string> List of tile urls.
+    const std::vector<std::string> getTiles();
```

Comment:
Should be marked `const`.

Reply:
I removed the type from the docstring. The return type already is marked `const` or am I missing something?

Reply:
Right, but the method itself can also be marked const right? It does not modify the object.

Reply:
ah yes. I don't use c++ too often and forgot about that. fixed it now.